desktop/cmp/form/FormField.js

```diff
@@ -0,0 +1,89 @@
+/*
+ * This file belongs to Hoist, an application development toolkit
+ * developed by Extremely Heavy Industries (www.xh.io | info@xh.io)
+ *
+ * Copyright © 2018 Extremely Heavy Industries Inc.
+ */
+import React, {Component} from 'react';
+import {elemFactory, HoistComponent} from '@xh/hoist/core';
+import {PropTypes as PT} from 'prop-types';
+import {HoistInput} from '@xh/hoist/cmp/form';
+import {formGroup, spinner} from '@xh/hoist/kit/blueprint';
+import {fragment, div, span} from '@xh/hoist/cmp/layout';
+import {throwIf} from '@xh/hoist/utils/js';
+import {isArray} from 'lodash';
+
+import './FormField.scss';
+
+/**
+ * Standardised wrapper around HoistInputs.
+ *
+ * Should receive a single HoistInput as a child element. If the HoistInput is bound
+ * to a model enhanced by `@FieldSupport`, FormField will automatically display a label,
+ * a required asterisk and validation state.
+ *
+ * Accepts any props supported by Blueprint's FormGroup.
+ */
+@HoistComponent
+export class FormField extends Component {
+
+    static propTypes = {
+        /** model to bind to */
+        model: PT.object,
+        /** name of property in model to bind to */
+        field: PT.string,
+        /** optional label for form field */
+        label: PT.string
+    };
+
+    baseClassName = 'xh-form-field';
+
+    render() {
+        const {model, field, ...rest} = this.props,
+            item = this.prepareChild(),
+            hasFieldSupport = model && field && model.hasFieldSupport,
+            fieldModel = hasFieldSupport ? model.getField(field) : null,
+            isRequired = fieldModel && fieldModel.isRequired,
+            isPending = fieldModel && fieldModel.isValidationPending,
+            notValid = fieldModel && fieldModel.isNotValid,
+            errors = fieldModel ? fieldModel.errors : [],
+            labelStr = this.props.label || (fieldModel ? fieldModel.displayName : null),
+            label = isRequired ? div(labelStr, span(' *')) : div(labelStr);
```

Comment:
Any sense we might want to customize or suppress this as the "required" hint?

I think the asterisk is pretty common, although it sometimes looks a bit meh. 

Just raising the question in case we have any ideas - *totally fine* with starting here and seeing if we are ever unhappy with it in practice.